src/routes/manga/+page.svelte

```diff
@@ -0,0 +1,108 @@
+                            <Circle class="{index === 2 && "hidden"} md:w-[0.35vw] opacity-50" />
+                        {/each}
+                    </manga-other-details>
+                    <genres class="flex items-center md:gap-[0.75vw]">
+                        {#each manga.genres as genre}
+                            <genre class="leading-none bg-white/10 backdrop-blur border-[0.01vw] border-white/25 md:px-[0.65vw] md:py-[0.35vw] md:rounded-[0.25vw] md:text-[0.9vw]">{genre}</genre>
+                        {/each}
+                    </genres>
+                    <options class="flex md:mt-[2.5vw] items-center md:gap-[1vw]">
+                        <button class="btn md:gap-[0.25vw] rounded-xl bg-warning-400 text-[3vw] font-bold text-surface-900 md:rounded-[0.5vw] md:text-[0.875vw] leading-none p-4 md:p-[1vw]">
+                            <Read class="md:w-[1.25vw]" />
+                            <span class="md:text-[1vw] font-semibold">Read Now</span>
+                        </button>
+                        <button class="btn md:gap-[0.25vw] rounded-xl bg-surface-900 text-[3vw] font-bold text-surface-50 md:rounded-[0.5vw] md:text-[0.875vw] leading-none p-4 md:p-[1vw]">
+                            <Info class="md:w-[1.25vw]" />
+                            <span class="md:text-[1vw] font-semibold">Details</span>
+                        </button>
+                        <button class="btn rounded-xl bg-surface-900 text-[3vw] font-bold text-surface-50 md:rounded-[0.5vw] md:text-[0.875vw] leading-none p-4 md:p-[1vw]">
+                            <Edit
+                                variant="with_underline_around_pencil"
+                                class="w-4 text-surface-50 md:w-[1.25vw]"
+                            />
+                        </button>
+                    </options>
+                </manga-info>
+                <manga-cover class="flex md:h-[25vw] shrink-0">
+                    <ImageLoader src={manga.cover} class="h-full w-full md:rounded-[0.5vw]" />
+                </manga-cover>
+            </manga>
+
+            <slider-buttons>
+                <button
+                    class="btn btn-icon absolute -left-[1vw] top-1/2 -translate-y-1/2 z-20 hidden h-[2.25vw] w-[2.25vw] rounded-[0.375vw] bg-secondary-800 md:flex"
+                >
+                    <Chevron
+                        color="text-white"
+                        class="w-[1.25vw] rotate-90"
+                    />
+                </button>
+                <button
+                    class="btn btn-icon absolute -right-[1vw] top-1/2 -translate-y-1/2 z-20 hidden h-[2.25vw] w-[2.25vw] rounded-[0.375vw] bg-secondary-800 md:flex"
+                >
+                    <Chevron
+                        color="text-white"
+                        class="w-[1.25vw] -rotate-90"
+                    />
+                </button>
+            </slider-buttons>
+        </slider>
+    </landing>
```

Comment:
![image](https://github.com/tokitou-san/CoreProject-Frontend/assets/61817579/f4cdc6cb-52b8-4c96-b6cf-9216524206f7)

we shouldn't have this large thingy

Reply:
huh! what :\
It looks really cool, whats the issue?

Reply:
>  It looks really cool, whats the issue?

Not more animes. Add nhentai

Reply:
> Not more animes. Add nhentai

Dont want this kinda section?
direct cards?